src/SFA.DAS.Payments.Monitoring.Metrics.Data/DcMetricsDataContext.cs

```diff
@@ -268,14 +390,30 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
         public async Task<List<TransactionTypeAmounts>> GetEarnings(long ukprn, short academicYear, byte collectionPeriod, CancellationToken cancellationToken)
         {
+            if (academicYear == 2526)
+            {
+                using (await BeginTransaction(cancellationToken))
+                {
+                    return await Earnings.FromSqlRaw(BaseDcEarningsQuery2526 + UkprnFilterSelect, new SqlParameter("@ukprn", ukprn), new SqlParameter("@collectionperiod", collectionPeriod)).ToListAsync(cancellationToken);
+                }
+            }
+
             using (await BeginTransaction(cancellationToken))
```

Comment:
Needs an else statement to prevent the 2425 query being run on the 2526 database, I think